Translations/translation_JA_JP.json

```diff
@@ -7,22 +7,22 @@
     "SettingsCalibrationDone": "校正完了",
     "SettingsCalibrationWarning": "続行する前に、コテ先が室温と同じになっていることを確認してください",
     "SettingsResetWarning": "設定をリセットしますか？",
-    "UVLOWarningString": "電圧が低すぎます",
-    "UndervoltageString": "Undervoltage",
-    "InputVoltageString": "Input V: ",
-    "WarningTipTempString": "Tip temp: ",
-    "BadTipString": "BAD TIP",
+    "UVLOWarningString": "DC電圧が低すぎます",
```

Comment:
Warning messages does not scroll so only 8 characters at maximum can be shown.

(I had thought of whether warning messages should be made scrolling instead, but non-CJK languages currently use small text in two rows instead of large text, so it's not as straightforward to change.)